.env.test

```diff
@@ -1,4 +1,5 @@
+DELETION_ENABLED=true
 ENCRYPTION_KEY=+WxrxtmkNdODyHrrwh1K2msWts1HVCCf7B98Q0odcUs=
+ORGANISATION_ID='local-testing'
```

Comment:
Did we add this org id to aws gateway? Otherwise, should probably change to 'local-development'. Not as pressing ofc but would still like to have a working example for testing

Reply:
This was actually a brainfart and can be removed

Reply:
Feexed